app/ai/voice/agents/breeze_buddy/services/telephony/plivo/conference.py

```
@@ -1,135 +1,141 @@
             )
-            if hasattr(response, "__dict__"):
-                logger.info(f"[Transfer] response.__dict__: {response.__dict__}")
 
             return {
                 "success": True,
-                "agent_call_uuid": customer_call_sid,
                 "reason": "success",
+                "agent_added": True,
             }
 
         except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)